core/src/ledger_cleanup_service.rs

```diff
@@ -13,15 +12,22 @@ use std::thread;
 use std::thread::{Builder, JoinHandle};
 use std::time::Duration;
 
-pub const DEFAULT_MAX_LEDGER_SLOTS: u64 = 3 * DEFAULT_SLOTS_PER_EPOCH;
+// This is chosen to allow enough time for
+// - To try and keep the RocksDB size under 128GB at 50k tps (100 slots take ~2GB).
+// - A validator to download a snapshot from a peer and boot from it
+// - To make sure that if a validator needs to reboot from its own snapshot, it has enough slots locally
+//   to catch back up to where it was when it stopped
+pub const MAX_LEDGER_SLOTS: u64 = 6400;
```

Comment:
@mvines is this fine?

Reply:
How did we come up with this `6400` number?

Reply:
Sure, seems like the best value we can come up with now